user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/Visual_Overview_component.md

```diff
@@ -26,3 +26,19 @@ To configure this component:
 >
 > - Spectrum components are currently not yet supported in visual overviews within dashboards or low-code apps.
 > - Quick filters are supported for table parameters in visual overview components. See [Using quick filters](xref:Using_quick_filters).
+
+## Zooming
+
+Zooming functionalities are available for the Visual Overview component.
+
+- From DataMiner 10.3.0 [CU18]/10.4.0 [CU6]/10.4.9 onwards<!--RN 40017-->:
+
+  - To zoom in, press CTRL while scrolling up.
+
+  - To zoom out, press CTRL while scrolling down.
+
+- Prior to DataMiner 10.3.0 [CU18]/10.4.0 [CU6]/10.4.9:
```

Comment:
Zooming was not mentioned yet on the Visual Overview component page, but I assume zooming in/out used to work by using the scroll wheel on its own instead of pressing CTRL at the same time. Let me know if this isn't correct, @Robin-Debel.

Reply:
Correct